### PR TITLE
Make a clear explanation of what the repo does at the start of the readme

### LINKED ISSUE #505: Make a clear explaination of what the repo does at the start of the readme
Docs-searchbar is a great tool if you know what it is. Arriving on the repo you are not met with explanation on what the plugin does. 

Taking inspiration of [`instant-meilisearch`](https://github.com/meilisearch/instant-meilisearch/) an explanation should be added at the top of the repo.

### PR DESCRIPTION
**Pull request by TechSpiritSS**
# Pull Request

## Related issue
Fixes #505 

## What does this PR do?
- I have added a quick intro in the beginning of README.MD as stated in the issue #505 

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,11 @@
...
 
-**docs-searchbar.js** is a front-end SDK for **Meilisearch** to handle your search bar documentation.
+Meilisearch is an open-source smart search engine that helps you find what you're looking for faster. **docs-searchbar.js** is a front-end SDK for **Meilisearch** to handle your search bar documentation. This makes it easy to integrate and get started with Meilisearch and focus on your acutal product rather than spenidng hours on making search faster.The goal of this library is to provide a front-end search bar into your own documentation. To make that possible, you need to gather your website content in advance, and index it in a Meilisearch instance. We have all the tools necessary for the job and you can [try out our playground](./CONTRIBUTING.md#playground). 
 
-**Meilisearch** is an open-source search engine. [Discover what Meilisearch is!](https://github.com/meilisearch/meilisearch)
+docs-searchbar.js comes with a css template. The default behavior of this library fits perfectly for a documentation search bar, but you can customize it according to your needs easily with few lines of code.
+
+[Discover what Meilisearch is!](https://github.com/meilisearch/meilisearch)
```
Comment (bidoubiwa) [inline README.md:27]:
```suggestion
**Meilisearch** is an open-source search engine. [Discover what Meilisearch is!](https://github.com/meilisearch/meilisearch)
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
Hey! Thanks a lot for the contribution :) 

Some changes are required

**Comment by TechSpiritSS** [comment]
Sure, I am on my way to improve this.

**Review by TechSpiritSS — COMMENTED** [review]
Fixed them as stated.

File: README.md [hunk 2]
```diff
@@ -20,11 +20,15 @@
...
+**docs-searchbar.js** is a front-end SDK for **Meilisearch** to handle your search bar documentation. The goal of this library is to provide a front-end search bar into your own documentation.
 
-docs-searchbar.js comes with a css template. The default behavior of this library fits perfectly for a documentation search bar, but you can customize it according to your needs easily with few lines of code.
+- Meilisearch is a RESTful search API. It aims to be a ready-to-go solution for everyone who wants a fast and relevant search experience for their end-users ⚡️🔎
```
Comment (bidoubiwa) [inline README.md:25]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -20,11 +20,15 @@
...
 
-[Discover what Meilisearch is!](https://github.com/meilisearch/meilisearch)
+**docs-searchbar.js** comes with a css template. The default behavior of this library fits perfectly for a documentation search bar, but you can customize it according to your needs easily with few lines of code.
+
+To get started head over to our [documentaion](https://docs.meilisearch.com/). To make it work, You need to have your documentation's content in a Meilisearch instance. If not already the case, you can achieve this using [`docs-scraper`](https://github.com/meilisearch/docs-scraper/).
```
Comment (bidoubiwa) [inline README.md:29]:
```suggestion
To make it work, You need to have your documentation's content in a Meilisearch instance. If not already the case, you can achieve this using [`docs-scraper`](https://github.com/meilisearch/docs-scraper/).
```

File: README.md [hunk 4]
```diff
@@ -20,11 +20,15 @@
...
+**docs-searchbar.js** is a front-end SDK for **Meilisearch** to handle your search bar documentation. The goal of this library is to provide a front-end search bar into your own documentation.
 
-docs-searchbar.js comes with a css template. The default behavior of this library fits perfectly for a documentation search bar, but you can customize it according to your needs easily with few lines of code.
+- Meilisearch is a RESTful search API. It aims to be a ready-to-go solution for everyone who wants a fast and relevant search experience for their end-users ⚡️🔎
 
-[Discover what Meilisearch is!](https://github.com/meilisearch/meilisearch)
+**docs-searchbar.js** comes with a css template. The default behavior of this library fits perfectly for a documentation search bar, but you can customize it according to your needs easily with few lines of code.
```
Comment (bidoubiwa) [inline README.md:27]:
```suggestion
**docs-searchbar.js** comes with a css template. The default styling of this library fits a documentation search bar, but you can customize it.
```

File: README.md [hunk 5]
```diff
@@ -20,11 +20,15 @@
   <a href="https://ms-bors.herokuapp.com/repositories/11"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
 </p>
 
-Meilisearch is an open-source smart search engine that helps you find what you're looking for faster. **docs-searchbar.js** is a front-end SDK for **Meilisearch** to handle your search bar documentation. This makes it easy to integrate and get started with Meilisearch and focus on your acutal product rather than spenidng hours on making search faster.The goal of this library is to provide a front-end search bar into your own documentation. To make that possible, you need to gather your website content in advance, and index it in a Meilisearch instance. We have all the tools necessary for the job and you can [try out our playground](./CONTRIBUTING.md#playground). 
+**docs-searchbar.js** is a front-end SDK for **Meilisearch** to handle your search bar documentation. The goal of this library is to provide a front-end search bar into your own documentation.
```
Comment (bidoubiwa) [inline README.md:23]:
```suggestion
**docs-searchbar.js** is a front-end SDK for **Meilisearch** providing a search bar for your documentation.
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

**Review by bidoubiwa — APPROVED** [review]
bors merge

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
Thanks again for contributing with Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive a small gift from Meilisearch too, don't forget to fill [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**